org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/model/GatewayBranch.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright 2017 ZTE Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ * Contributors:
+ *     ZTE - support of more gateways
+ *******************************************************************************/
+package org.eclipse.winery.bpmn2bpel.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class represents a sub path of a gateway node
+ * 
+ * it contains two properties, id is the first following node's id; 
+ * nodeList is the nodes of the branch order by flow sequence.
+ *
+ */
+public class GatewayBranch {
+	
+	protected String id;
```

Comment:
Yeah, it should be private. I overlook that.

However, what i meant is very simple: There seem spaces be used instead of tabs. See the output of GitHub at https://github.com/ZTE/winery/blob/538eb3734768c417bc2ca5f38fed7e2e18b3ac45/org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/model/GatewayBranch.java

![grafik](https://cloud.githubusercontent.com/assets/1366654/23786414/3864806e-056d-11e7-988c-6b1ff63686ce.png)